packages/third-parties/src/ThirdPartyScriptEmbed.tsx

```diff
@@ -15,6 +17,13 @@ export default function ThirdPartyScriptEmbed({
   children,
   dataNtpc = '',
 }: ScriptEmbed) {
+  useEffect(() => {
+    // Useful for feature detection and measurement
+    performance.mark('next-third-parties', {
```

Comment:
We have to use the special `mark_use_counter` name so it's picked up by Chrome tooling. The unique name should be in the details field.

```
performance.mark('mark_use_counter',
    {'detail': {
        'feature': 'next-third-parties'
        }
    }
);

```